src/main/java/org/isf/operation/gui/OperationRowEdit.java

```diff
@@ -401,6 +397,32 @@ private JComboBox getOperationsBox() {
 		comboOpe.setEnabled(true);
 		return comboOpe;
 	}
+	private JComboBox getComboResultBox() {
+		JComboBox comboResult = new JComboBox();
+			for (String description : operationResults) {
+				comboResult.addItem(description);
+			}
+			if(opeRow != null){
```

Comment:
I know the existing code had no spaces before `(` often (e.g. `if(`') but the current project standards include the space, `if (`.   It would be great if when adding/changing code those changes where made at the same time.  Thanks